lib/any.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var R = require('ramda');
+
+/**
+ * Output a block (or its inverse) based on whether or not any of the supplied
+ * arguments are truthy.
+ *
+ * @since v0.11.0
+ * @param {...*} values One or more values to test against.
+ * @param {Object} options
+ * @return {String|Boolean}
+ * @example
+ * var a = true;
+ * var b = 0;
+ * var c = false;
+ *
+ * {{#any a b}}✔︎{{else}}✘{{/all}} //=> ✔︎
+ * {{#any b c}}✔︎{{else}}✘{{/all}} //=> ✘
+ * 
+ * {{#if (any a b)}}
+ *   Also works inline!
+ * {{/if}}
+ */
+
+function any() {
+  var options = R.last(arguments);
+  var result = false;
+  var values;
+  var i;
+  
+  if (R.isNil(options.fn)) {
+    options = undefined;
+    values = arguments;
+  } else {
+    values = R.dropLast(1, arguments);
+  }
+  
+  for (i = 0; i < values.length; i++) {
```

Comment:
@tylersticka I was `console.log`ging the values for my own learning purposes and I noticed a detail I wanted to ask about. 🙂 

For the inline helper test:
```js
template = Handlebars.compile('{{#if (all a b c)}}✔︎{{/if}}');
actual = template({ a: true, b: true, c: true });
```
The values are as follows:
```js
{ 
  '0': true,
  '1': true,
  '2': true,
  '3': { name: 'all', hash: {}, data: { root: [Object] } } 
}
```
Is this expected? I see `a`, `b` and `c` _plus_ a fourth value as well. 🤔 

The tests _pass_ but I was still curious. 😉

Reply:
@gerardo-rodriguez Can you elaborate on what you were logging and where?

Reply:
@tylersticka Sure thing!

```diff
 lib/all.js | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lib/all.js b/lib/all.js
index bb1bbca..d907030 100644
--- a/lib/all.js
+++ b/lib/all.js
@@ -36,6 +36,8 @@ function all() {
     values = R.dropLast(1, arguments);
   }
 
+  console.log('values', values);
+
   for (i = 0; i < values.length; i++) {
     if (!values[i]) {
       result = false;
```
🙂

Reply:
@gerardo-rodriguez Thank you!

In this case, no, that should _not_ still be in the array.

It _believe_ it used to be that Handlebars would only pass along a final object argument when a helper was used in block syntax. Perhaps that's changed.

Looks like I've got some additional cleanup to do! Good catch.

Reply:
@tylersticka Oops, LOL

I meant to attach [my original comment](https://github.com/cloudfour/core-hbs-helpers/pull/71/files#r308441819) from this thread to the `lib/all.js` file. My bad! 😅